test/functional/authorization.spec.ts

```diff
@@ -160,7 +160,7 @@ describe("NodeJS CRUD Tests", function() {
       const { resource: successDoc } = await col1Client
         .database(entities.database.id)
         .container(entities.coll1.id)
-        .item(entities.doc1.id)
+        .item(entities.doc1.id, EMPTY_PARTITION_KEY)
```

Comment:
let's just use `undefined` instead of this random constant?

Reply:
For all cases in our tests/samples

Reply:
Agreed